RequestManagementSystem/Client/Request.py

```diff
@@ -167,8 +167,8 @@ def _notify( self ):
       # # All operations Done -> Done
       elif opStatus == "Done" and self.__waiting == None:
         rStatus = "Done"
-        self._Error = ''
-    self.Status = rStatus
+        self.Error = ''
+    self._Status = rStatus
```

Comment:
no
self.Status = rStatus

Have a look at what are "properties" in Python, this is it :-)